### PR TITLE
feat: add create-react-native-app spec

### PR DESCRIPTION
**Pull request by EvanBacon**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

Adds support for `create-react-native-app`. Utilizes the icons added for https://github.com/withfig/autocomplete/pull/419 (I maintain both tools).

### CODE REVIEW

**Comment by EvanBacon** [comment]
Heads up, external contributors cannot interact with checkboxes on comments from this bot.

File: dev/create-react-native-app.ts [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+        isOptional: false,
+      },
+      icon: ICONS.template,
+      priority: 76,
+    },
+    {
+      name: ["--template-path"],
+      description: "The path inside of a GitHub repo where the example lives.",
+      args: {
+        name: "name",
```
Comment (Monkeyank) [inline dev/create-react-native-app.ts:60]: Add `template: "folders"`

**Review by Monkeyank — CHANGES_REQUESTED** [review]
Thanks so much for this spec! Just a little nit!

File: dev/create-react-native-app.ts (hunk 1 above)
Comment (QuiiBz) [reply dev/create-react-native-app.ts:60]: I don't think that this template is appropriated. Based on the description, the path refers to the github repo, not the user's computer.

**Review by QuiiBz — APPROVED** [review]
LGTM!

**Review by cstrnt — APPROVED** [review]

**Pull request merged**